libs/domain/cart/list-item/list-item.component.ts

```diff
@@ -0,0 +1,117 @@
+import { CartComponentMixin, CartService, PriceMode } from '@spryker-oryx/cart';
+import { resolve } from '@spryker-oryx/di';
+import { I18nMixin, Size } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+import { CartListItemProperties } from './list-item.model';
+import { cartListItemStyles } from './list-item.styles';
+
+const size = Size.Md;
+
+export class CartListItemComponent
+  extends CartComponentMixin(I18nMixin(LitElement))
+  implements CartListItemProperties
+{
+  static styles = cartListItemStyles;
+
+  protected cartService = resolve(CartService);
+
+  @property({ type: Boolean, reflect: true }) open?: boolean;
+
+  protected setDefault(): void {
+    this.cartService.setDefaultCart({ cartId: this.$cart()!.id }).subscribe();
+  }
+
+  protected override render(): TemplateResult | void {
+    const cart = this.$cart();
+
+    if (!cart) return;
+
+    const totalQuantity = this.$totalQuantity() ?? 0;
+    //TODO: use link service or content link
+    const url = `/my-account/carts/${cart.id}`;
+
+    return html`
+      <oryx-collapsible ?open=${this.open}>
+        <oryx-link slot="heading">
+          <a href=${url}>
+            ${cart.name}
+            ${this.i18n('cart.cart.totals.<count>-items', {
+              count: totalQuantity,
+            })}
+          </a>
+        </oryx-link>
+        ${when(
+          cart.isDefault,
+          () =>
+            html`<oryx-chip slot="heading" appearance="success">
+              ${this.i18n('default')}
+            </oryx-chip>`
+        )}
+
+        <oryx-site-price
+          slot="heading"
+          .value=${cart.totals?.priceToPay ?? 0}
+          .currency=${cart.currency}
+        ></oryx-site-price>
+
+        <span slot="heading"
+          >(
+          ${this.i18n(
+            cart.priceMode === PriceMode.GrossMode
+              ? 'cart.mode.gross'
+              : 'cart.mode.net'
+          )}
+          )</span
+        >
+
+        ${when(
+          !totalQuantity,
+          () => html`<p>${this.i18n('cart.list.no-cart-entries')}</p>`,
+          () => html`
+            <oryx-cart-entries cartId=${cart.id}></oryx-cart-entries>
+          `
+        )}
```

Comment:
Can we only render this when the collapsible is open? It means that we won't load entries while they're not needed, which saves us from initial loading.

Reply:
Currently is not possible, collapsible does not emit any event to check changing of the `open` state. But we can provide oryx.toggle event that will be emitted on collapsible toggle, up to you

Reply:
yes, but we don't know when collapsible (who control the state) is changing it